### ISSUE TITLE
Pretrained weights

### ISSUE THREAD

**Issue by mat02** [MADONOKOUKI/Block-wise-Scrambled-Image-Recognition#6]
Hi,
could you provide a link to download pretrained checkpoint?
I'm trying to reproduce your results, but on CIFAR10 I cannot achieve more than ~20% accuracy.

**Comment by MADONOKOUKI** [comment]
I didn't provide any pre-trained model. but you can reproduce the results.
I'm so sorry but can you tell me the details of your training settings? I might help your concerns.

**Comment by mat02** [comment]
Sorry for such a late reply.
I was finally reproduce results of your experiments, but only after carefully recreating the same environment. It appears that installing newer PyTorch version and it's dependencies somehow broke the training process.

**Comment by MADONOKOUKI** [comment]
I don't know your training settings in a details but it has no problem on my side
